src/validators/problem.ts

```diff
@@ -40,4 +42,10 @@ export function problemValidator(method: string): ValidationChain[] {
             ];
         }
     }
+}
+
+async function validateProblemSetIds(problemSetIds: Array<string>) {
+    const promiseArray = problemSetIds.map(async problemSetId => await problemSetDBInteractions.find(problemSetId));
```

Comment:
logic here is to map each problemSetId into a Promise<IProblemSetModel> object, wait for those promises to resolve and then make sure none of the resolved values are null.